src/components/media/MediaBody.js

```diff
@@ -0,0 +1,22 @@
+// @flow
+import * as React from 'react';
+import classnames from 'classnames';
+import './Media.scss';
+
+type Props = {
+    /** Child elements */
+    children: React.Node,
+    /** Additional class names */
+    className?: string,
+};
+
+function MediaBody(props: Props) {
```

Comment:
For code consistency I would do

```jsx
const MediaBody = ({ className, children, ...rest }: Props) => (
        <div className={classnames('bdl-Media-body', className)} {...rest}>
            {children}
        </div>
);
```

For example even within this PR two functional components seem to have their props extracted differently. And then most of our functional components normally use the const foo = () style.